agot-bg-game-server/src/client/game-state-panel/ResolveSingleMarchOrderComponent.tsx

```diff
@@ -140,6 +146,16 @@ export default class ResolveSingleMarchOrderComponent extends Component<GameStat
                             <Form.Group as={Col}>
                                 <Form.Label>
                                     Do you want to leave a Power token in <b>{startingRegion.name}</b> to keep control?
+                                    {this.canLeavePowerToken || this.canLeavePowerTokenReason == "no-power-token-available" ?
```

Comment:
That will result in the warning being displayed only if the user is out of Power tokens. Maybe you can modify it to show it if the decision is `No` and hide the warning as long as it is `undefined`?